Wikipedia/Code/SavedArticlesFetcher.m

```diff
@@ -177,14 +177,14 @@ - (void)fetchAllImagesInArticle:(MWKArticle*)article failure:(WMFErrorHandler)fa
 
     NSArray<NSURL*>* URLs = [[article allImageURLs] allObjects];
     
-    [self.imageController cacheImagesWithURLsInBackground:URLs failure:failure success:success];
+    [self cacheImagesWithURLsInBackground:URLs failure:failure success:success];
 }
 
 - (void)fetchGalleryDataForArticle:(MWKArticle*)article failure:(WMFErrorHandler)failure success:(WMFSuccessHandler)success {
     WMF_TECH_DEBT_TODO(check whether on - disk image info matches what we are about to fetch)
     @weakify(self);
     [self fetchImageInfoForImagesInArticle:article failure:^(NSError *error) {
-        
+        failure(error);
```

Comment:
this might have been missing before the conversion away from promise kit. seems like it should be called, but dunno if it has unintended consequences.